cmd/root.go

```diff
@@ -13,5 +19,47 @@ var rootCmd = &cobra.Command{
 }
 
 func Execute() {
-	rootCmd.Execute()
+	if err := rootCmd.Execute(); err != nil {
+		log.Fatal(err)
+		os.Exit(-1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+}
+
+func initConfig() {
+	// set up ENV var support
+	viper.SetEnvPrefix("pflt")
+	viper.AutomaticEnv()
+
+	// set up optional config file support
+	viper.SetConfigName("config")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			log.Info("Config file not found.  Proceeding without it.")
+		}
+	}
+
+	// Set up logging config defaults
+	viper.SetDefault("logfile", "preflight.log")
+	viper.SetDefault("loglevel", "trace")
```

Comment:
I feel like the default log level should actually be something much lower, like warning. Trace is the most verbose. I feel like only a developer would need trace.

Reply:
Agreed.  Changed the default to warn.